paddle/fluid/eager/auto_code_generator/final_state_generator/eager_gen.py

```diff
@@ -213,8 +213,12 @@ def ParseYamlReturns(string):
 
     returns = [x.strip() for x in string.strip().split(",")]
     for i in range(len(returns)):
-        ret = returns[i]
-        returns_list.append(["", ret, i])
+        ret_type = returns[i]
```

Comment:
If `ret_type` is a string like `Tensor(out)`，can it be found in `yaml_types_mapping.keys()`?